src/Runner.Worker/ActionManager.cs

```diff
@@ -464,6 +547,121 @@ private async Task BuildActionContainerAsync(IExecutionContext executionContext,
             }
         }
 
+        // todo: Remove when feature flag DistributedTask.NewActionMetadata is removed
```

Comment:
This will be removed when the REST call to the service is implemented, right? It's only called from the temporary implementation below.

Reply:
Yea I went through and left comments on all the places where "legacy" code needs to be removed when we delete the feature-flag.

Reply:
Oops yes i need to fix the comment:

```
// This implementation is temporary and will be replaced with a REST API call to the service to resolve
```